testsuite/tests/libratest/smoke_test.rs

```diff
@@ -930,6 +930,72 @@ fn test_e2e_reconfiguration() {
     );
```

Comment:
Perhaps a question for later, but: should we add more test files instead of letting smoke_test grow to 1K+ lines?